backend/protocol_rpc/endpoints.py

```
@@ -49,6 +49,11 @@
 from backend.protocol_rpc.message_handler.types import LogEvent, EventType, EventScope
 from backend.protocol_rpc.types import DecodedsubmitAppealDataArgs
 from backend.database_handler.snapshot_manager import SnapshotManager
+from datetime import datetime
+import time
+from web3 import Web3
+import rlp
+import backend.node.genvm.origin.calldata as calldata
```

Ruff (0.11.9)

`backend.node.genvm.origin.calldata` imported but unused

Remove unused import: `backend.node.genvm.origin.calldata`

(F401)